### PR TITLE
Header styling

### PR DESCRIPTION
**Pull request by Chaseknis**
Added Index.html
Added style.css
Added test.md
Added .gitignore 
fixed end-of-source-newline error
edited the readme file
provided the pull request description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,285 @@
...
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Felix45) [inline README.md]: Good job adding the right `readme` template to your project this is very commendable :clap: :clap: , however you forgot to customize the content on the readme file to your project. Please fill in the different sections of the readme file with details about your project.

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (Felix45) [inline test.md]: - [ ] Please make sure that this file is added to the `.gitignore` file and that it's not present in your GitHub repository. Please note that this is part of the project requirements as shown in the screenshot below.

Screenshot from 2022-11-30 11-22-31

You can do this by un-tracking the file in `git` using the following command

```
git rm -r --cached test.md
```
Once the file is un-tracked, add the file to the `.gitignore` file and commit the changes using the `git commit -m` command.

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @Chaseknis , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Highlights :heavy_check_mark: 
- [ ] Github flow is correctly used :white_check_mark: 
- [ ] Good commit messages :white_check_mark: 


 ## Required Changes ♻️
 - [ ] Please provide a meaningful summary of this pull request. The summary should include all the changes that were introduced in this pull request so that someone can understand what the pull request is about. Currently, the pull request has no summary.
 
Screenshot from 2022-11-30 11-15-50


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Chaseknis) [reply README.md]: @Felix45 Thank you for the feedback.
I have made all the necessary changes hoping to hear from you soon.

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Chaseknis   👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Chaseknis** [comment]
Thank you @nedjwab

File: README.md (hunk 1 above)
Comment (Felix45) [reply README.md]: Good job, keep coding :+1:

**Pull request merged**